eXtendingHEEP.md

```diff
@@ -0,0 +1,309 @@
+# Extending X-HEEP via the eXtension interface
```

Comment:
actually the CGRA is not extended with the X interface but it is memory mapped, so I suggest you say "Extending X-HEEP" without specifying how